sklearn/utils/__init__.py

```diff
@@ -471,9 +471,3 @@ def tosequence(x):
         return list(x)
```

Comment:
Your earlier comment was wrt to the `ConvergenceWarning` and `DataDimensionalityWarning` which were removed from this file...